prow/config.yaml

```diff
@@ -89,6 +89,12 @@ slack_reporter_configs:
       *Commit:* <{{.BaseLink}}|{{printf "%.7s" .BaseSHA}}>
       {{end}}*<{{.Status.URL}}|View logs>*
 
+in_repo_config:
+  enabled:
+    "*": true
+  allowed_clusters:
+    "*": ["default", "untrusted-workload", "trusted-workload"]
```

Comment:
In a secret with workload clusters kubeconfig a default context is pointing to untrusted-workloads. This is not a prow cluster.
```
contexts:
- context:
    cluster: untrusted-workload
    user: untrusted-workload
  name: default
```

Reply:
Right, I've assumed it wrong. Then why do you duplicate this name? We don't need to use `default` here then.

Reply:
We need to use it in case anyone will use it in it's prowjob definition. With this list we allow inrepo prowjobs to run on all contexts we have defined in Prow,